src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -121,6 +121,7 @@ void SolverInterfaceImpl::configure(
     PRECICE_INFO("This is preCICE version " << PRECICE_VERSION);
     PRECICE_INFO("Revision info: " << precice::preciceRevision);
     PRECICE_INFO("Configuring preCICE with configuration: \"" << configurationFileName << "\"");
+    PRECICE_INFO("I am participant: \"" << _accessorName << "\"");
```

Comment:
Using both colon and quotes reads a bit weird.

```suggestion
    PRECICE_INFO("I am participant \"" << _accessorName << "\"");
```

Reply:
Agree. I also changed the previous line (configuration file).